chrome/content/vendor/jasmine/reporters/terminal_reporter.js

```diff
@@ -253,7 +253,7 @@
             for(i = 0; i < color_attributes.length; i++) {
                 ansi_string += "\033[" + ATTRIBUTES_TO_ANSI[color_attributes[i]] + "m";
             }
-            ansi_string += string + "\033[" + ATTRIBUTES_TO_ANSI["off"] + "m";
+            ansi_string += string + "\033[" + ATTRIBUTES_TO_ANSI.off + "m";
```

Comment:
We do not maintain the vendor libraries, so this change should not be included in the PR.  The best place to make this change is upstream.